src/test/java/com/fasterxml/jackson/databind/ConstructorDetector4860Test.java

```diff
@@ -0,0 +1,47 @@
+package com.fasterxml.jackson.databind;
```

Comment:
Let's not add tests are main package level -- this is creators-related. I'll move it